ucm2/Toradex/verdin/dahlia-HiFi.conf

```diff
@@ -0,0 +1,35 @@
+# Use case configuration for Toradex Dahlia Carrier Board
+
+SectionDevice."Headphone" {
+	Comment "Headphone"
+
+	BootSequence [
```

Comment:
Overall: BootSequence must be in top configuration files.

Reply:
Fixed, thanks!